src/test/java/com/powsybl/openloadflow/sensi/ac/AcSensitivityAnalysisContingenciesTest.java

```diff
@@ -605,11 +605,12 @@ void testContingencyPropagationLfSwitch() {
 
         SensitivityAnalysisResult result = sensiRunner.run(network, factors, contingencies, Collections.emptyList(), sensiParameters);
 
+        // FIXME: contingency propagation is deactivated for the moment.
```

Comment:
@geofjamg I have temporary deactivated the contingency propagation for sensitivity analysis. I am going to switch the sensitivity analysis on the bus/breaker view in an other PR. But, if it is too dangerous for you, I can introduce a parameter on the sensitivity analysis in this PR and the propagation will be done on the bus/view, with all the problems that you know.